tests/tools/test_ruff_works.py

```diff
@@ -59,10 +59,10 @@ def test_ruff_works(
         f"{expected_path_0}:cell_1:1:8: F401 [*] `os` imported but unused\n"
         f"{expected_path_0}:cell_1:3:8: F401 [*] `glob` imported but unused\n"
         f"{expected_path_0}:cell_1:5:8: F401 [*] `nbqa` imported but unused\n"
-        f"{expected_path_0}:cell_4:1:1: E402 Module level import not at top of file\n"
-        f"{expected_path_0}:cell_4:1:20: F401 [*] `random.randint` imported but unused\n"
-        f"{expected_path_0}:cell_5:1:1: E402 Module level import not at top of file\n"
-        f"{expected_path_0}:cell_5:2:1: E402 Module level import not at top of file\n"
+        f"{expected_path_0}:cell_4:3:1: E402 Module level import not at top of file\n"
+        f"{expected_path_0}:cell_4:3:20: F401 [*] `random.randint` imported but unused\n"
+        f"{expected_path_0}:cell_5:3:1: E402 Module level import not at top of file\n"
+        f"{expected_path_0}:cell_5:4:1: E402 Module level import not at top of file\n"
```

Comment:
this isn't quite right...somehow, need to sort out the line number in the sub too